src/library.js

```diff
@@ -27,12 +27,12 @@ LibraryManager.library = {
 
   getTempRet0__sig: 'i',
   getTempRet0: function() {
-    return {{{ makeGetTempRet0() }}};
+    return getTempRet0();
   },
 
   setTempRet0__sig: 'vi',
-  setTempRet0: function($i) {
-    {{{ makeSetTempRet0('$i') }}};
+  setTempRet0: function(val) {
+    setTempRet0(val);
```

Comment:
```suggestion
    setTempRet0(val | 0);
```
With this, this is not NFC, unless you verified all the callers.

Reply:
Oh, I thought from our conversation about that the `| 0` stuff was no longer needed so could just be skipped.  I didn't do any kind of verification so I guess I need to restore this.

Reply:
My understanding is that all callers of these `_` prefixed versions should be native code .. there are no callers that I know of in JS that use the `_` prefixed versions.. they all call the support.js versions AFAICT.

Reply:
(I don't see any myself.)